ui/components/component-library/modal-focus/index.ts

```diff
@@ -0,0 +1,2 @@
+export { ModalFocus } from './modal-focus';
+export type { ModalFocusProps } from './modal-focus.types';
```

Comment:
Will be removed once https://github.com/MetaMask/metamask-extension/pull/18979 is merged